efi/meson.build

```diff
@@ -226,6 +229,7 @@ app = custom_target(efi_name,
                       '--arch', gnu_efi_arch,
                       '--os', host_machine.system(),
                       '--objcopy', objcopy,
+                      '--objcopy-manualsymbols', objcopy_manualsymbols,
```

Comment:
by making this argument boolean, you can add it only in the cases needed instead